src/core/modules/slide/service/slide.service.js

```diff
@@ -1,20 +1,42 @@
+import connection from 'core/database';
+import { errorResponse, failResponse, successResponse } from 'package/handler/bot-response';
+import { logger } from 'package/logger';
+
 class SlideServiceImpl {
+    constructor() {
+        this.db = connection;
```

Comment:
You should create a repository layer, such as slide.repository.js, the repository class is responsible for querying database